README.md

```diff
@@ -607,7 +607,7 @@ highlight QuickScopePrimary guifg='#afff5f' gui=underline ctermfg=155 cterm=unde
 highlight QuickScopeSecondary guifg='#5fffff' gui=underline ctermfg=81 cterm=underline
 ```
 
-to your init.vim. The underline color can be changed by the `guisp` tag.
+to your init.vim. The underline color can be changed by the `guifp` tag.
```

Comment:
why is minutiae about random plugins, in the readme at all? It's complete noise.

Reply:
The plan was for information like that to move to the wiki - it's included at all because it's useful enough information not to disappear in an old issue thread.